examples/service-whoami/whoami-oauth.py

```diff
@@ -17,7 +17,9 @@
 from jupyterhub.utils import url_path_join
 
 class WhoAmIHandler(HubOAuthenticated, RequestHandler):
-    hub_users = {getuser()} # the users allowed to access this service
+    # hub_users is a set of users who are allowed to access the service
+    # `getuser()` here means only the user who started the service
+    # can access the service:
```

Comment:
Did you mean to delete `hub_users = {getuser()}`?

Reply:
oops, no. Good catch.